### ISSUE TITLE
how to install on mac

### ISSUE THREAD

**Issue by littlebeandog** [Norbyte/bg3se#37]
how to install on mac

**Comment by Rukongai** [comment]
Mac doesn't even have a full release of BG3 yet - it's slated for later this month.

If you are playing using GPT or Crossover - use winetricks.

Outside of that, you'll have to wait and hope

**Comment by CorneliusMH** [comment]
We have a full release now! My guess is that the location will be:

/Users/username/Library/Application Support/Steam/steamapps/common/Baldurs Gate 3/Baldur's Gate 3.app/Contents/Frameworks

As that's where the bink libraries are for the mac version. I dumped the contents of my Baldurs Gate 3.app folder: [bg3macContents.txt](https://github.com/Norbyte/bg3se/files/12707192/bg3macContents.txt)

However, looks like the libraries there are .dylib files and not .dlls so it might not be so easy.

**Comment by CorneliusMH** [comment]
Apparently, norbyte confirmed that the DOS2 version of the script extender would never work with mac so I fear that that may be the case again. Which is a bummer, but not the end of the world for me, a person who can use the non-mac version as well.

**Comment by ArkaneScythe** [comment]
It probably can be re coded entirely from scratch as a mac version but that would be ALOT of work, and from what I can tell no one has yet or is in the process of

**Comment by eirinym** [comment]
It would be fantastic if there were to be a way to install it on Mac. If it can work on Linux, a Mac option would be feasible. If there weren't several good mods which require script extender to function, it wouldn't be such an important factor. I do hope it can be done.

**Comment by Rukongai** [comment]
> It would be fantastic if there were to be a way to install it on Mac. If it can work on Linux, a Mac option would be feasible. If there weren't several good mods which require script extender to function, it wouldn't be such an important factor. I do hope it can be done.

This isn't accurate. It works on Linux because the Linux version of the game is running through WINE - emulating a windows platform - meaning that windows based modifications of the game can be made to work (specifically by setting a DLL override in this instance)

The macOS version is native. It is an entirely separate beast and SE would more or less have to be rebuilt from scratch.

**Comment by mdecrocemovson** [comment]
any update on this? I have mac and it sounds like there's no chance of it working. is there?

**Comment by Norbyte** [comment]
tl;dr there are currently no plans for Mac version.

**Comment by jackCat101** [comment]
Documents/Baldur's Gate 3/bin
